.github/workflows/yaml_and_ansible_linting.yml

```diff
@@ -0,0 +1,33 @@
+name: "Run ansible-lint and yamllint for this repo."
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push events and pull_request
+  #push:
+  pull_request:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a set of commands using the runners shell
+      - name: "Ansible-lint"
+      # Install ansible-lint together with yamllint and run the linting on the whole repo
+        run: |
+          sudo pip install "ansible-lint[yamllint]"
```

Comment:
Can you pin version of packages from pypi?

Reply:
Sure thing.

Reply:
Is that format sufficient, our you want to set each one individually?